### PR TITLE
SDIT-2123: 🔊 Reduce redis sampling to 1% and change logging to info level

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: applicationinsights.dev.json [hunk 1]
```diff
@@ -6,29 +6,60 @@
     "service.version": "${BUILD_NUMBER}"
   },
   "instrumentation": {
-    "springScheduling": {
```
Comment (petergphillips) [inline applicationinsights.dev.json:9]: not needed these days

File: applicationinsights.dev.json [hunk 2]
```diff
@@ -6,11 +6,8 @@
...
-      "enabled": false
-    },
     "logging": {
-      "level": "DEBUG"
+      "level": "INFO"
```
Comment (petergphillips) [inline applicationinsights.dev.json:10]: not interested in debug logging for this app

File: applicationinsights.dev.json [hunk 3]
```diff
@@ -19,16 +16,49 @@
...
+      {
+        "telemetryType": "dependency",
+        "attributes": [
+          {
+            "key": "db.system",
+            "value": "redis",
+            "matchType": "strict"
+          }
+        ],
+        "percentage": 1
```
Comment (petergphillips) [inline applicationinsights.dev.json:28]: redis set to 1% as nothing interesting

File: applicationinsights.dev.json [hunk 4]
```diff
@@ -19,16 +16,49 @@
...
+      },
+      {
+        "telemetryType": "dependency",
+        "attributes": [
+          {
+            "key": "server.address",
+            "value": "sts.eu.*",
```
Comment (petergphillips) [inline applicationinsights.dev.json:35]: the sqs traffic isn't that exciting

File: helm_deploy/offender-events-ui/values.yaml [hunk 5]
```diff
@@ -12,8 +12,6 @@ generic-service:
 
   ingress:
     enabled: true
-    v1_2_enabled: true
```
Comment (petergphillips) [inline helm_deploy/offender-events-ui/values.yaml:15]: remove old unused properties

File: helm_deploy/offender-events-ui/values.yaml [hunk 6]
```diff
@@ -26,12 +24,6 @@ generic-service:
     SPRING_PROFILES_ACTIVE: "logstash"
     SPRING_DATA_REDIS_SSL_ENABLED: "true"
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY);IngestionEndpoint=https://northeurope-0.in.applicationinsights.azure.com/;LiveEndpoint=https://northeurope.livediagnostics.monitor.azure.com/"
-    HMPPS_SQS_USE_WEB_TOKEN: "true"
```
Comment (petergphillips) [inline helm_deploy/offender-events-ui/values.yaml:29]: default is `true` these days anyway

**Review by lbennett-moj — APPROVED** [review]

**Pull request merged**